src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -280,7 +281,7 @@ internal static string GetDocumentXmlPath(XmlDocument document)
         /// to be saved to the XmlDocument. This parameter cannot be null and 
         /// must represent a non-empty list of node-data-list pairs.</param>
         internal static void SaveTraceDataToXmlDocument(XmlDocument document,
-            IEnumerable<KeyValuePair<Guid, List<string>>> nodeTraceDataList)
+            IEnumerable<KeyValuePair<Guid, List<CallSite.RawTraceData>>> nodeTraceDataList)
```

Comment:
Are these methods used by DynamoRevit? If yes then we might be breaking binary compatibility.